site/src/components/series/boxPlot.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: component
+title: Box Plot Series
+component: series/boxPlot.js
+tags:
+  - playground
+namespace: series
+
+example-code: |
+  //Generating the boxPlot information for the data
+  data.forEach(function(d) {
+      d.yUpBox = Math.random();
+      d.yDownBox = Math.random();
+      d.yUpWhisker = d.yUpBox + Math.random();
+      d.yDownWhisker = d.yDownBox +  Math.random();
+  });
+
+  var boxPlot = fc.series.boxPlot()
+      .xScale(xScale)
+      .yScale(yScale)
+      .boxLow(function(d,i) { return d.close - d.yDownBox; })
+      .boxHigh(function(d,i) { return d.close + d.yUpBox; })
+      .whiskerLow(function(d,i) { return d.close - d.yDownWhisker; })
+      .whiskerHigh(function(d,i) { return d.close + d.yUpWhisker; });
+
+  container.append('g')
+      .datum(data)
+      .call(boxPlot);
+
+---
+
+A [box plot series](https://en.wikipedia.org/wiki/Box_plot) is a convenient way of graphically depicting groups of 
+numerical data through their quartiles. Boxes can be renderer vertically or horizontally based on the value of the `orient` property.
+
+The upper and lower end of each box are defined by the `boxUpper` and `boxLower` properties.
+The upper and lower whisker of each box are defined by the `whiskerUpper` and `whiskerLower` properties.
```

Comment:
It's probably worth noting that when the bar is vertical the box / whisker properties are in the yScale coordinate system, whereas when it is horizontal, they are in the xScale coordinate system.